sdk/nodejs/examples/pastebin/src/index.ts

```diff
@@ -0,0 +1,90 @@
+const { Command } = require("commander");
+const zrok = require("@openziti/zrok")
+var readlineSync = require('readline-sync');
+
+
+const program = new Command();
+
+program
+  .command('copyto')
+  .version("1.0.0")
+  .description("command to host content to be pastedfrom'd")
+  .action(async () => {
+    var data = readlineSync.question('Input some text... ');
+
+    console.log("data is: ", data)
+
+    var enc = new TextEncoder();
+    var buf = enc.encode(data); //
+
+    zrok.setLogLevel(0)
+
+    let root = zrok.Load()
+    await zrok.init( root ).catch(( err: Error ) => { console.error(err); return process.exit(1) });
+    console.log("setting up zrok.CreateShare...")
+    let shr = await zrok.CreateShare(root, new zrok.ShareRequest(zrok.TCP_TUNNEL_BACKEND_MODE, zrok.PRIVATE_SHARE_MODE, "pastebin", ["private"]));
```

Comment:
Same thing here... there is no public frontend called `private`... this list of public frontends should either be `null` or `[]`.

Reply:
fixed